src/phoenix/trace/fixtures.py

```diff
@@ -225,7 +254,7 @@ class TracesFixture:
 NAME_TO_TRACES_FIXTURE: Dict[str, TracesFixture] = {
     fixture.name: fixture for fixture in TRACES_FIXTURES
 }
-PROJ_NAME_TO_TRACES_FIXTURE = defaultdict(list)
+PROJ_NAME_TO_TRACES_FIXTURE: Dict[str, List[TracesFixture]] = defaultdict(list)
```

Comment:
fyi. defaultdict has a type
```suggestion
PROJ_NAME_TO_TRACES_FIXTURE: DefaultDict[str, List[TracesFixture]] = defaultdict(list)
```